litestar/channels/memory.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import annotations
```

Comment:
I'd rename the file to `in_memory_backend.py`

Reply:
See comment below